BetSystem/Validator/BetOnEventValidator.cs

```diff
@@ -8,11 +7,14 @@ public class BetOnEventValidator : AbstractValidator<BetOnEventDto>
     {
         public BetOnEventValidator()
         {
-            
+
             RuleFor(x => x.BetOnResult).IsInEnum().WithMessage("Put 0-Win or 1-Lose or 2-Draw");
-            RuleFor(x => x.Amount).NotNull();
-            RuleFor(x => x.Interest).NotNull();
-            RuleFor(x => x.AmountToPay).NotNull();
+            RuleFor(x => x.Amount).NotNull().NotEmpty();
+            RuleFor(x => x.Interest).NotNull().NotEmpty();
+            RuleFor(x => x.AmountToPay).NotNull().NotEmpty();
```

Comment:
What if it will be negative int - it's discount or we may need validator for that?

Reply:
Done